### DISCUSSION TITLE
IPED 4.1.x python plugins for Windows

### DISCUSSION THREAD

**Discussion by lfcnassif** [sepinf-inc/IPED#1822] [General]
Hello all,

Since some users are having issues to install IPED extra python packages on Windows, I decided to distribute a pre-built package. I included all libraries needed to enable face recognition and audio transcription using Facebook Wav2Vec2, on CPU. Libraries needed to enable the alternative Yahoo NSFW nudity detection are **not** included. Just download the package below, unzip it and replace the whole IPED's python folder for its contents:
https://github.com/sepinf-inc/IPED/releases/download/4.1.4/IPED-4.1.x_python_plugins_Win64_CPU_only.zip

To enable face recognition, set `enableFaceRecognition = true` into `IPEDConfig.txt`.

To use Facebook Wav2Vec2 for audio transcription, you must set `enableAudioTranscription = true` into `IPEDConfig.txt` and go to `conf/AudioTranscriptConfig.txt` and uncomment (remove starting \#) this line:
`implementationClass = iped.engine.task.transcript.Wav2Vec2TranscriptTask`

You also must uncomment **one**  `huggingFaceModel` into that same file, like:
`huggingFaceModel = Edresson/wav2vec2-large-xlsr-coraa-portuguese` (faster)
`huggingFaceModel = jonatasgrosman/wav2vec2-xls-r-1b-portuguese` (better)

You can find models for other languages there.

Best regards,
Luís Nassif

**Comment by gfd2020** [comment]
I am using this stand alone python to install the necessary dependencies for iped. When I try to install packages with pip, it gives an error because it seems that the path is absolute with the path 'E:\git\iped\target\release\iped-4.1.4\python\python.exe'. The only way to resolve this was to create a virtual disk with that same path. Is there another way to resolve it?

**Reply by lfcnassif** [reply]
Hi @gfd2020. Yes, this absolute path issue is described in the wiki manual, it's a pip behavior. You just need to install it again with `python get-pip.py`

**Reply by gfd2020** [reply]
Hi @lfcnassif . I'm sorry, I really missed that part in the manual. Thank you for your help.
